pipeline/experiment.py

```diff
@@ -277,6 +312,62 @@ class ActionEvent(dj.Imported):
     action_event_time : decimal(8,4)  # (s) from trial start
     """
 
+
+# ---- Foraging paradigm specifics ----
+
+@schema
+class SessionBlock(dj.Imported):
+    definition = """
+    -> Session
+    block : smallint 		# block number
+    ---
+    block_start_time : decimal(10, 4)  # (s) relative to session beginning
+    """
+
+    class WaterPortRewardProbability(dj.Part):
```

Comment:
It's minor but I think the WaterPort is unnecessary in the part table name.
It is just simply the RewardProbability of the Block and WaterPort must be defined inside the table.
Unless we would like to define another RewardProbablilty that is independent of the waterport, which I don't think.